Dockerfile

```diff
@@ -11,9 +11,9 @@ RUN echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.
 RUN apt-get update
 
 WORKDIR /opt
-COPY google-cloud-sdk /opt/google-cloud-sdk
+COPY google-cloud-sd[k] /opt/google-cloud-sdk
```

Comment:
Why the `[k]` here?  I don't see `google-cloud-sdk` in the tree still, so I don't think that this works.

Reply:
I was trying to work around not having conditional copies.